subprojects/core/src/integTest/groovy/org/gradle/api/CrossBuildScriptCachingIntegrationSpec.groovy

```diff
@@ -614,18 +614,6 @@ task fastTask { }
 
     def "script don't get recompiled if daemon disappears"() {
         root {
-            buildSrc {
```

Comment:
It didn't seem to have anything to do with what's being tested. We reverted the change because it's unrelated to script plugins so it's confusing to be in this PR.